### DISCUSSION TITLE
Config item  Question

### DISCUSSION THREAD

**Discussion by goerdi** [WGDashboard/WGDashboard#543] [General]
Hi !
I cannot find the part in the Doc which desribes th items 
[Server]
app_prefix = 

and

[WireGuardConfiguration]
autostart = 

Ok the last one could be autostart of wireguard (if its not done by the system but the first one...

And how is the configuration that im able to switch wireguard of with the button in top right of the GUI if i do not use autostart option ?

Ciao Gerd

**Comment by donaldzou** [comment]
Hi! `app_prefix` is for if your deployment of WGDashboard is under some path, i.e not at the domain's top level.

**Comment by goerdi** [comment]
hi  !

so that mean if i put in here wg-dashboard/ or /wg-dashboard and i have the rigcht config in apache... i can call it via
host.domain.tld/wg-dashboard/ ?

Ciao Gerd

**Reply by donaldzou** [reply]
Hi! I don't think that's how it works... It has to be configured with reverse proxy

**Reply by goerdi** [reply]
like in the message below?

What you see in the message is the result when i configure  it with apache like below... (i also have on another machine wireguard UI running and there it works like this => preifx and then through an apache reverse proxy..

Ciao Gerd

**Reply by goerdi** [reply]
HI !

OK  im one step far.....

wg dashboard is running on (e.g.)
http://192.168.99.1:10086/wireguard/#/
now i want to have it at
http://host.domain.tld/wireguard

i got it running as  host.domain.tld through reverse apache proxy

**Comment by goerdi** [comment]
HI !

Ok i tried all possible variants and i always get 
-----
<div class="toolbar"><div class="devtools-separator"></div><div class="devtools-searchbox"><input class="searchBox devtools-filterinput" placeholder="JSON durchsuchen" value=""></div></div><div class="panelContent" id="json-scrolling-panel" tabindex="0">
  |  
-- | --
data | null
message | "Unauthorized access."
status | false
</div>
-----
the setting in apache is

    <Location /wireguard>#
	DirectorySlash On
	Order allow,deny
	Allow from all
	ReWriteEngine on
	ProxyPreserveHost on
	ProxyPass http://127.0.0.1:10086/
	ProxyPassReverse http://127.0.0.1:10086/
    </Location>
i also tried some variants in apache2

Ciao Gerd
